platform/model/typical_queries/hits.go

```diff
@@ -241,7 +244,8 @@ func (query Hits) computeIdForDocument(doc model.SearchHit, defaultID string) st
 			// At database level we only compare timestamps with millisecond precision
 			// However in search results we append `q` plus generated digits (we use q because it's not in hex)
 			// so that kibana can iterate over documents in UI
-			pseudoUniqueId = fmt.Sprintf("%xq%s", vv, defaultID)
+			sourceHash := fmt.Sprintf("%x", ComputeHash(doc.Source))
+			pseudoUniqueId = fmt.Sprintf("%xqqq%sqqq%x", vv, defaultID, sourceHash)
```

Comment:
What is this `defaultID`?

Reply:
Shouldn't we delete it.

Reply:
https://github.com/QuesmaOrg/quesma/pull/1435/commits/e8cc635cc7645e090e522f3393f760b8f21753dd

Reply:
Ok, will still moving to next doc work after your change?